### PR TITLE
Release 1.1.0933.01 to Insider channel

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
Release 1.1.0933.01 to Insider channel. Product changes are outlined in the release notes

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: Channels/Insider/release_info.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "default": {
-    "current_version": "1.1.0883.01",
-    "minimum_version": "1.1.0883.01",
-    "installer_url": "https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0883.01/AccessibilityInsights.msi",
-    "release_notes_url": "https://www.github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0883.01"
+    "current_version": "1.1.0933.01",
+    "minimum_version": "1.1.0933.01",
+    "installer_url": "https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.08933.01/AccessibilityInsights.msi",
```
Comment (DaveTryon) [inline Channels/Insider/release_info.json]: Typo
```suggestion
    "installer_url": "https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0933.01/AccessibilityInsights.msi",
```

File: Channels/Insider/release_notes.md [hunk 2]
```diff
@@ -1,28 +1,27 @@
-## June 4th, 2019 Insider Release ([v1.1.0883.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0883.01))
+## July 24 2019 Insider Release ([v1.1.0933.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0933.01))
```
Comment (DaveTryon) [inline Channels/Insider/release_notes.md]: Add comma
```suggestion
## July 24, 2019 Insider Release ([v1.1.0933.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0933.01))
```

File: Channels/Insider/release_notes.md [hunk 3]
```diff
@@ -1,28 +1,27 @@
-## June 4th, 2019 Insider Release ([v1.1.0883.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0883.01))
+## July 24, 2019 Insider Release ([v1.1.0933.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0933.01))
 
-Welcome to the June 4th, 2019 release of Accessibility Insights for Windows.
-
-Installation Link: [https://github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0883.01/AccessibilityInsights.msi](https://github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0883.01/AccessibilityInsights.msi)
-
-Documentation Link: [https://accessibilityinsights.io/docs/en/windows/overview](https://accessibilityinsights.io/docs/en/windows/overview)
+Welcome to the July 24 2019 Insider release of Accessibility Insights for Windows.
```
Comment (DaveTryon) [inline Channels/Insider/release_notes.md]: Add comma
```suggestion
Welcome to the July 24, 2019 Insider release of Accessibility Insights for Windows.
```

File: Channels/Insider/release_notes.md [hunk 4]
```diff
@@ -1,28 +1,27 @@
...
-
-The core scanning code from Accessibility Insights for Windows has been moved into a separate GitHub repo (https://github.com/microsoft/axe-windows). Accessibility Insights now gets its core scanning code from that location. This code will be handled separately to make this functionality more broadly available to the accessibility community.
+  - [UIAccess changes](#rule-updates)
+  - [Bug Fixes](#bug-fixes)
+  - [Rule Updates](#rule-updates)
 
-#### Telemetry Improvements
+#### UIAccess Changes
 
-We identified a few points in the code where we had no telemetry about failures that users might be experiencing. We added these data points to help us make the product better. No additional customer data is exposed.
```
Comment (Sirisha-Kommushetty) [inline Channels/Insider/release_notes.md]: Let's remove "No additional customer data is exposed"

**Comment by DaveTryon** [comment]
> @DaveTryon - Do take a look at my single comment before submission.

@Sirisha-Kommushetty, the telemetry comment was from the previous set of release notes and is being removed. Proceeding with release.

**Pull request merged**